kaggle_tools_update.Dockerfile

```diff
@@ -18,8 +18,10 @@ FROM gcr.io/kaggle-images/python:latest
 
 RUN pip install --upgrade git+https://github.com/Kaggle/learntools
 
-# Set up pip to enable pip install.
-ADD patches/kaggle_bashrc /root
-# Patch the system-wide bashrc file for non-root users.
-RUN cat /root/kaggle_bashrc >> /etc/bash.bashrc
-RUN rm /root/kaggle_bashrc
+# TODO(dsjang): Remove these lines once the docker image build turns green since they are copied from Dockerfile
+# to apply on top of the last green.
+# Set up an initialization script to be executed before any other commands initiated by the user.
+ADD patches/entrypoint.sh /root/entrypoint.sh
+RUN chmod +x /root/entrypoint.sh
+# This script gets executed by "docker run <image> <command>" and it runs <command> at the end of its execution. 
+ENTRYPOINT ["/root/entrypoint.sh"]
```

Comment:
I'm not sure if it's just me (Vincent did this as well), but the default (on Linux anyhow) I'm accustomed to is to have a final newline at the EOF (for a text file).

Reply:
VSCode doesn't add a one by default. I changed my settings after last time